### ISSUE TITLE
Make selecting bones nicer

### ISSUE THREAD

**Issue by stockto2** [stockto2/desktophero#91]
Some possible options: 
1) Use color picking for bone selecting, if three.js lets you define color for separate bones
2) Create a bounding box around the bones by adding a child and seeing where the child is located in world space (end of the bone) and creating a box around it, then adding that to the bone. Use color picking to select the bounding boxes.

**Comment by swordsmanr** [comment]
I have noticed that when bone groups are close togeter, it's hard to select the right one - selecting neck vs head, or weapon vs. hand - is that was this task is referring to?

**Comment by stockto2** [comment]
Correct. It would be nice to be able to click anywhere on the bone to select it rather than on the base of the bone.
